ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml

```diff
@@ -698,6 +698,10 @@ tests:
       OPENSHIFT_INSTALL_RELEASE_IMAGE: release:latest-multi
     env:
       ANSIBLE_PLAYBOOK_CREATE_INFRA: equinix_heterogeneous_create_infra_playbook.yml
+      ASSISTED_CONFIG: |
+        AGENT_DOCKER_IMAGE=quay.io/edge-infrastructure/assisted-installer-agent-multi:latest
```

Comment:
Actually, I think we can build the prow images multi-arch in the first place by reverting this:
https://github.com/openshift/release/pull/34041

It should be better for performance and for having the jobs more portable to other branches

Reply:
I think the problem was that the registry is not multi-arch. We have one URI for x86 and one for arm, which is not what I want here.

Reply:
I see
Then looks good to me